plumbing/revlist/revlist.go

```diff
@@ -16,7 +16,26 @@ import (
 // the reachable objects from the given objects. Ignore param are object hashes
 // that we want to ignore on the result. All that objects must be accessible
 // from the object storer.
-func Objects(s storer.EncodedObjectStorer, objects, ignore []plumbing.Hash) ([]plumbing.Hash, error) {
+func Objects(
+	s storer.EncodedObjectStorer,
+	objs []plumbing.Hash,
+	ignore []plumbing.Hash) ([]plumbing.Hash, error) {
```

Comment:
They are not duplicated. The public signature is still the same. However, I moved most of the logic of `Objects` to a private implementation `objects` which can be called with further arguments. That one is used from `Objects` and it's called twice.